content/en/docs/application-store/built-in-apps/etcd-app.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "etcd App"
+keywords: 'kubernetes, kubesphere, etcd, app-store'
+description: 'How to use etcd'
+
+
+weight: 2240
+---
+
+# Objective
+
+This tutorial shows you how to quickly deploy an [etcd](https://etcd.io) application using templates from KubeSphere App Store (Powered by OpenPitrix). The demonstration includes importing application repository, sharing and deploying apps within a workspace.
+
+# Prerequisites
+
+- You have enabled [KubeSphere Application Store](/docs/pluggable-components/app-store/)
+- You have completed the tutorial [Create Workspace, Project, Account and Role](/docs/quick-start/create-workspace-and-project/). The account needs to be a platform regular user and to be invited as the project operator with the `operator` role. In this tutorial, we'll work in the project `apps` of the workspace `apps`.
+
```

Comment:
It is better to use project-apps/ns-apps and ws-apps for the names respectively.

Reply:
done